package.json

```diff
@@ -139,6 +139,7 @@
         ]
     },
     "dependencies": {
+        "@guardian/src-foundations": "^3.12.0",
```

Comment:
We should be using the `@guardian/source-foundations` peerDependency

Reply:
It is already a peer dep. I think this import was auto added by mistake by VSCode - I really need to make it stop doing that.

Removed now 👍